src/sentry/integrations/vercel/webhook.py

```diff
@@ -1,24 +1,135 @@
+import hashlib
```

Comment:
This file has not been changed but was previously `generic_webhook.py`. It replaced the previous `webhook.py` which is why this diff is so detailed. The only change was adopting the name `VercelGenericWebhookEndpoint` -> `VercelWebhookEndpoint`